### ISSUE TITLE
Can not set team associated with resource item via API

### ISSUE THREAD

**Issue by QB-SZ** [elabftw/elabapi-python#22]
### Details
I create a resource item via the API. I am part of two teams. It seems by default the resource is assigned to the first team. I cannot change the team via API or the web interface afterwards.
### Expected behaviour
Option to set team id upon creation of resource via API or option to change team association later on via patch_item()
### Elab Version
5.0.0

**Comment by NicolasCARPi** [comment]
Hello,

This is not an API issue, the software doesn't allow changing the `team` attribute currently. This will need to be added in eLab first.

In the meantime, you can use different api keys from different teams (as keys are bound to a team).

**Comment by QB-SZ** [comment]
Hi, you are very fast today :) 
So does this feature request exist? It would be of high importance to us. Also should I close this issue then?

**Comment by NicolasCARPi** [comment]
AFAIK it doesn't exist yet, you should create it.

**Comment by mhxion** [comment]
> This will need to be added in eLab first.

+1 for the feature. It is not an immediate issue for us, but there were times when we also needed to modify different "entities" of multiple teams, and we had to use individual API keys.

<details open>
<summary> More info: </summary>
Our case was more tricky, because we also needed to auto-create a number of teams, so we (mainly @alexander-haller) had to modify the DB directly to auto-generate the API keys for those teams! In terms of this approach, one way to "enable" this feature would be: implementing a way to <i>create</i> API keys for any given team, and not just the for the team the calling member is tied to.
</details>
